<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote Memory Scanner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableWidget" name="tableWidgetSearchResults">
           <property name="font">
            <font>
             <family>Courier New</family>
            </font>
           </property>
           <column>
            <property name="text">
             <string>Address</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Previous</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="labelFound">
             <property name="text">
              <string>Found: 0</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelDisplayIfMoreThan">
             <property name="text">
              <string>Display if less than</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxDisplayIfLessThan">
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelSelectedProcess">
           <property name="text">
            <string>No Process Selected</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelIteration">
           <property name="text">
            <string>Iteration: 0</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButtonFirstScan">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>First Scan</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonNextScan">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Next Scan</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacerScan">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonUndoScan">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Undo Scan</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="labelValue">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Value</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditValue">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelScanType">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Scan Type</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBoxScanType">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>Exact Value</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelValueType">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Value Type</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="comboBoxValueType">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>Integer</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxMemoryScanOptions">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="title">
            <string>Memory Scan Options</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QCheckBox" name="checkBoxMappedModulesOption">
              <property name="text">
               <string>Include Mapped Modules</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacerOptions">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonRefreshFoundValues">
           <property name="text">
            <string>Refresh found values</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTableWidget" name="tableWidgetAddresses">
       <property name="font">
        <font>
         <family>Courier New</family>
        </font>
       </property>
       <column>
        <property name="text">
         <string>Description</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Address</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
       </column>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>689</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenProcess"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpenProcess">
   <property name="text">
    <string>Open Process</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
